actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -181,9 +180,15 @@ private void getUploadStatus(Request actorMessage) {
         }
         sender().tell(response, self());
       } else {
-        response = new Response();
-        response.put(
-            JsonKey.RESPONSE, "Operation is still in progress, Please try after some time.");
+        resMap.put(JsonKey.PROCESS_ID, resMap.get(JsonKey.ID));
+        if ((int) resMap.get(JsonKey.STATUS)
```

Comment:
Can you move the if check to updateResponseStatus?

Pass the int status to this method where the switch can be there for COMPLETED, IN_PROGRESS etc.